src/amber/cli/templates/app/.amber.yml.ecr

```diff
@@ -2,3 +2,27 @@ type: app
 database: <%= @database %>
 language: <%= @language %>
 model: <%= @model %>
+watch:
+  server: # required: the first command for this task is blocking
+    files:
+      - "src/**/*.cr"
+      - "src/**/*.<%= @language %>"
+      - "config/**/*.cr"
+    commands:
+      - "crystal build -o bin/<%= @name %> src/<%= @name %>.cr -p --no-color"
+      - "bin/<%= @name %>"
+
+  client: # required: these files changes trigger browser reloading
```

Comment:
should this be required? I generally do almost no front end code, what if I don't want watch to do anything?

Reply:
@robacarp You're right, actually this field can be optional, the current implementation can work without a `client` field